packages/experiments/src/components/Pagination/Pagination.types.ts

```diff
@@ -0,0 +1,117 @@
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunctionOrObject, IRefObject } from '../../Utilities';
+
+export interface IPagination {}
+
+export interface IPaginationProps {
+  /**
+   * Required: Selected index
+   */
+  selectedPageIndex: number;
```

Comment:
It feels odd that this prop is required. If you look across the OUFR project I don't think you will find any required props. Possible default to the 1st page if the consuming app doesn't keep a state of which page is selected.

Reply:
addressed. Made it optional and added default value

---
In reply to: [221845313](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r221845313) [](ancestors = 221845313)